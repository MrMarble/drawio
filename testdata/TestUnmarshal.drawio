<mxfile host="app.diagrams.net" modified="2022-04-10T09:33:09.085Z" agent="5.0 (X11)" etag="fC_GmiomcjAtfOR-YLAa" version="17.4.2" type="device"><diagram id="Z21_1_7TJZtvcwcqZcWw" name="Page-1">zVZNc5swEP01PrYDKCbkaDtOMplmevAh8VFBa1ArWEaWDfTXVxhhoWA7X85MTmjfrpD03u5KIzLLqltJi/QBGYhR4LFqRK5HQeCTywv9aZC6Ra580gKJ5MwEWWDB/4EBPYNuOIO1E6gQheKFC8aY5xArB6NSYumGrVC4qxY0gQGwiKkYoo+cqbRFo+DS4nfAk7Rb2Q+vWk9Gu2BzknVKGZY9iMxHZCYRVTvKqhmIhryOl3bezRHvfmMScvWWCQ8SJsvf83u4eQynjMzVc7n9MTZ7U3V3YGD6/MZEqVJMMKdibtGpRX8hFjrM1+AfUKo28tGNQg2lKhPGCxVXT3rs/Rwba9nzXDfZ4nVG3Rm5kvVT31jaPzSmnbazunkSNzkDZv4/ZMkQt8aNjOEENV22UZmAOhFn8rnhrbeA0eAWMAO9Ox0gQVDFt25eUZOeyT7OKqgHRsR3COoPBLV8NOyUKVewKOju6KWuWVeoo3RtQSqoTh7QeC9MvteuWdrq8buSSHuVE3pfxEhwghH/dUZWXIgZCpS7uYRRiFaxxtdK4l/oecI4gufVeTj8diSGH+gTLs2f6Bp+r2fYDnK4a3y83skb6z34VvVOhsIIoS9HeD2z6bpob8wVrxqZzpG4fugm7t7uZW50IHGjr0pcc09vqdgYfiZiSyUOaNMHVC4/boXnmMOLdmAgKniSazPWtIHGpw1dXD8hJsaRccbEMUHc9nwGCUj0oneMhxKEByQg75dAm/b9svP1XoFk/h8=</diagram></mxfile>