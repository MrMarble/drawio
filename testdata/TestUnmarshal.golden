<mxfile host="app.diagrams.net" modified="2022-04-10T09:33:09.085Z" agent="5.0 (X11)" etag="fC_GmiomcjAtfOR-YLAa" version="17.4.2" type="device">
  <diagram id="Z21_1_7TJZtvcwcqZcWw" name="Page-1">
    <mxGraphModel dx="1374" dy="913" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="827" pageHeight="1169" math="0" shadow="0">
      <root>
        <mxCell id="0"></mxCell>
        <mxCell id="1" parent="0"></mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-5" parent="1" source="MreAYOEJeFW6Bd3Etbwv-1" target="MreAYOEJeFW6Bd3Etbwv-3" edge="1">
          <mxGeometry as="geometry" relative="1"></mxGeometry>
        </mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-1" parent="1" vertex="1">
          <mxGeometry as="geometry" x="40" y="40" width="120" height="60"></mxGeometry>
        </mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-2" parent="1" vertex="1">
          <mxGeometry as="geometry" x="240" y="40" width="120" height="60"></mxGeometry>
        </mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-6" parent="1" source="MreAYOEJeFW6Bd3Etbwv-3" target="MreAYOEJeFW6Bd3Etbwv-2" edge="1">
          <mxGeometry as="geometry" relative="1"></mxGeometry>
        </mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-3" parent="1" vertex="1">
          <mxGeometry as="geometry" x="160" y="160" width="80" height="80"></mxGeometry>
        </mxCell>
        <mxCell id="MreAYOEJeFW6Bd3Etbwv-7" parent="1" value="Alvaro" vertex="1">
          <mxGeometry as="geometry" x="380" y="450" width="60" height="30"></mxGeometry>
        </mxCell>
      </root>
    </mxGraphModel>
  </diagram>
</mxfile>